packages/web-forms/src/components/widgets/RadioButton.vue

```diff
@@ -45,8 +45,13 @@ const setSelect1Value = (item: SelectItem) => {
   outline: 1px solid var(--surface-300);
 	border-radius: 10px;
   padding: 15px;
-	margin: 20px 0;
+	margin: 0 0 20px 0;
   cursor: pointer;
+	background: var(--surface-0);
+
+	&:last-of-type{
+		margin-bottom: 0;
```

Comment:
![image](https://github.com/user-attachments/assets/63873a0b-3d3e-4d21-9f49-74c598274bac)

I think I mentioned in #164 concerns about using `margin` for spacing. This is a really good example of why `gap` might be a better choice.

(I think I recall there was a fix for this previously, but I saw it in #164 as well, and now I see it's moved here.)